Gemfile.lock

```diff
@@ -204,3 +205,9 @@ DEPENDENCIES
   unicorn
   will_paginate_mongoid (~> 2.0)
   yajl-ruby
+
+RUBY VERSION
```

Comment:
We keep adding and removing this.  Is it fine to just leave this in to know how this was created?